docs/concepts/clickhouse.rst

```diff
@@ -0,0 +1,92 @@
+Clickhouse
+##################################
+
+ClickHouse is a high-performance, column-oriented SQL database management system (DBMS) 
+for online analytical processing (OLAP). It is used as the backend storage for the Aspects 
+project that powers Superset.
+
+Connect to Clickhouse Cloud
```

Comment:
I think this should also be a how-to linked to from here, and linked with a note that just says it can be run locally or remotely such as with ClickHouse Cloud.